homeassistant/components/yeelight/__init__.py

```diff
@@ -532,15 +555,21 @@ def async_unload(self):
 class YeelightEntity(Entity):
     """Represents single Yeelight entity."""
 
-    def __init__(self, device: YeelightDevice):
+    def __init__(self, device: YeelightDevice, unique_id: str):
         """Initialize the entity."""
         self._device = device
+        self._unique_id = f"v2-{unique_id}"  # Prefix to make migration easier
```

Comment:
If I set unique ID as device ID in config flow, can I use the config entry unique ID as entity unique ID and fall back to entry ID when that is not available? Then it will not actually change the unique IDs if the integration currently works.

Reply:
I'm confused by your question here?

The config flow unique ID doesn't have the same goal/function like the one in an entity.

Reply:
Oh okay. So can I use use the device id from entry data as unique ID and fall back to entry id if the device id does not exist?

Reply:
I fail to see how they are related?

Reply:
Currently entity unique IDs are based on device IDs. So if we can set entity IDs as device IDs when possible and fall back to entry ID then we do not need to migrate any entity unique IDs.